.github/workflows/makewindows.yml

```diff
@@ -0,0 +1,48 @@
+name: make windows
+
+on:
+  workflow_dispatch:
+    branches: [ $default-branch, 0.20, 0.21 ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install general dependencies
+      run: |
+        sudo apt update
+        sudo apt upgrade
+        sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git
+    - name: install the mingw-w64 cross-compilation tool chain
+      run: sudo apt install g++-mingw-w64-x86-64
+    - name: update compiler alternatives
+      run: echo "1" | sudo update-alternatives --config x86_64-w64-mingw32-g++ 
+    - name: make dependencies
+      run: |
+        cd depends
+        sudo chmod +x config.guess
+        sudo chmod +x config.site.in
+        sudo chmod +x config.sub
+        sudo make -j4 HOST=x86_64-w64-mingw32 
```

Comment:
This is a very cumbersome assembly and it never changes. Maybe it makes sense to put it in a separate job and use an artifact with a long lifetime?

Reply:
It's possible.  The x86_64-w64-mingw32 folder is 285mb so it will use a lot of the account storage limit when uploaded unless there is a place to wget it from.

Reply:
There are storage options in docker hub or on one of our cloud machines.